### PR TITLE
Key visibility

### PR DESCRIPTION
**Pull request by compnerd**

### CODE REVIEW

**Comment by compnerd** [comment]
CC @parkera

**Comment by compnerd** [comment]
@swift-ci please test

File: CoreFoundation/Locale.subproj/CFDateFormatter.c [hunk 1]
```diff
@@ -41,7 +41,7 @@ typedef CF_ENUM(CFIndex, CFDateFormatterAmbiguousYearHandling) {
 
 extern UCalendar *__CFCalendarCreateUCalendar(CFStringRef calendarID, CFStringRef localeID, CFTimeZoneRef tz);
 
-static CONST_STRING_DECL(kCFDateFormatterFormattingContextKey, "kCFDateFormatterFormattingContextKey");
+CONST_STRING_DECL(kCFDateFormatterFormattingContextKey, "kCFDateFormatterFormattingContextKey");
```
Comment (parkera) [inline CoreFoundation/Locale.subproj/CFDateFormatter.c]: Was this meant to switch to the PE version as below? Otherwise, why remove the static?

Comment (compnerd) [inline CoreFoundation/Locale.subproj/CFDateFormatter.c]: I noticed that they were used with externally visible functions, so it seems that the key could be used externally.

**Comment by compnerd** [comment]
ping?

CC: @phausler

File: CoreFoundation/Locale.subproj/CFDateFormatter.c (hunk 1 above)
Comment (parkera) [inline CoreFoundation/Locale.subproj/CFDateFormatter.c]: Sometimes these are used with the same functions but only meant to be used internally to CF.

Comment (compnerd) [inline CoreFoundation/Locale.subproj/CFDateFormatter.c]: Okay, Ill update this.

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by parkera** [comment]
@swift-ci please test and merge

**Pull request merged**